test/controllers/confirmations_controller_test.rb

```diff
@@ -2,7 +2,42 @@
 require "test_helper"
 
 class ConfirmationsControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  test "confirming with a valid token redirects to root page" do
```

Comment:
mmh je crois que j'aime mieux le given when then simplifié genre `given valid token, it should redirect to root page`

Reply:
J'allais dans l'idée qu'on lit la ligne au complet - `test that confirming...`. On peut essayer le given aussi

Reply:
ouin cest comme les it en jest j'ai ignorerait XD, avoir la pre condition au debut c'est vraiment plus clair je trouve

Reply:
Sure, ben là faut je me bouge faque je ferai pas le changement tout suite mais si tu veux faire ça + merge ça serait good